ads/google/doubleclick.js

```diff
@@ -79,6 +79,14 @@ function doubleClickWithGpt(global, data, gladeExperiment) {
     parseInt(data.overrideHeight || data.height, 10),
   ]];
 
+  // Center the ad in the container.
+  const container = global.document.querySelector('#c');
+  container.style.top = '50%';
+  container.style.left = '50%';
+  container.style.bottom = '';
+  container.style.right = '';
+  container.style.transform = 'translate(-50%, -50%)';
```

Comment:
This is the div from frame.html/remote.html set to position absolute with positions 0,0,0,0. Same for other comment.
https://github.com/ampproject/amphtml/blob/master/3p/remote.html